w4af/core/controllers/dependency_check/platforms/ubuntu1404.py

```diff
@@ -27,9 +27,6 @@
 class Ubuntu1404(Ubuntu1204):
     SYSTEM_NAME = 'Ubuntu 14.04'
 
-    CORE_SYSTEM_PACKAGES = Ubuntu1204.CORE_SYSTEM_PACKAGES[:]
```

Comment:
Same here. I don't see a justification for removing the `libjpeg-dev` dependency